packages/shopify-api/lib/session/types.ts

```diff
@@ -33,7 +33,22 @@ export interface SessionParams {
   /**
    * Information on the user for the session. Only present for online sessions.
    */
-  onlineAccessInfo?: OnlineAccessInfo;
+  onlineAccessInfo?: OnlineAccessInfo | StoredOnlineAccessInfo;
+}
+
+interface StoredOnlineAccessInfo {
```

Comment:
Should this `extend OnlineAccessInfo`?

Reply:
Is this what you were thinking?
https://github.com/Shopify/shopify-api-js/pull/1288/commits/f8be3ada25db2cd96fbbb6d968a86b6bc89f92a6